client/python/lib/test/config.py

```diff
@@ -0,0 +1,19 @@
+
+import pytest
+
+MODEL_REQUEST_VALID = [
+    ("model_name", 1),
+    ("3", 17),
+    ("*.-", 0),
+    ("model_name", 9223372036854775806),
+]
+
+MODEL_REQUEST_INVALID = [
+    (12, 1, pytest.raises(TypeError), "model_name should be type string, but is type int"),
+    (None, -1, pytest.raises(TypeError), "model_name should be type string, but is type NoneType"),
+    (None, 1, pytest.raises(TypeError), "model_name should be type string, but is type NoneType"),
+    ("model_name", "3", pytest.raises(TypeError), "model_version should be type int, but is type str"),
+    ("model_name" , None, pytest.raises(TypeError), "model_version should be type int, but is type NoneType"),
+    ("model_name", -1, pytest.raises(ValueError), "model_version should be positive integer, but is negative"),
+    ("model_name", 9223372036854775809, pytest.raises(ValueError), "model_version should have max 63 bits, but has 64"),
```

Comment:
The 3rd argument is in form `pytest.raises(<error_type>)`
Can we change it to just <error_type> and then in the test go with:
`with pytest.raises(exception) as e_info:`
It would give us more understanding of what is happening on the test level and keep just raw values in the test cases